### PR TITLE
Claim Deduction concepts and tutorial

### PR DESCRIPTION
**Pull request by bddap**
Requesting a once-over from either Lovesh or Sam.

### CODE REVIEW

File: tutorials/src/concepts_claim_deduction.md [hunk 1]
```diff
@@ -0,0 +1,169 @@
+# Claim Deduction
+
+The [verifiable credentials data model](https://www.w3.org/TR/vc-data-model/) is based on a machine comprehensible language called [RDF](https://www.w3.org/TR/rdf-primer/). RDF represents arbitrary semantic knowledge as [graph](https://en.wikipedia.org/wiki/Graph_(discrete_mathematics))s. Computers can perform automatic deductive reasoning over RDF, given assumptions (represented as an RDF graph) and axioms (represented as logical rules), a computer can infer new conclusions and even prove them to other computers using deductive derivations (proofs).
+
+So what does that have to do with verifiable credentials? Every VCDM credential is an RDF claim graph. Computers can reason about them, deriving new conclusions that weren't explicitly stated by the issuer.
+
+The dock javascript SDK exposes utilities for primitive deductive reasoning over verified credentials. The Verifier has a choice to perform deduction themself (expensive), or offload that responsibility to the Presenter of the credential[s] by accepting deductive proofs of composite claims.
```
Comment (SamHellawell) [inline tutorials/src/concepts_claim_deduction.md:7]: `The dock javascript SDK` -> `The Dock SDK`

Comment (SamHellawell) [reply tutorials/src/concepts_claim_deduction.md:7]: `over verified credentials` -> `over verifiable credentials`

**Review by SamHellawell — APPROVED** [review]
couple changes but lgtm for the most part

File: tutorials/src/concepts_claim_deduction.md (hunk 1 above)
Comment (bddap) [reply tutorials/src/concepts_claim_deduction.md:7]: Use of `verified` was intentional.

Comment (bddap) [reply tutorials/src/concepts_claim_deduction.md]: Changed `dock javascript SDK` to `Dock SDK` in https://github.com/docknetwork/sdk/pull/184/commits/e0b487b9fcade50451703813ce1912b9262ed5e2

**Comment by bddap** [comment]
Rendered:
- [concepts](https://docknetwork.github.io/sdk/tutorials/concepts_claim_deduction.html)
- [tutrorial](https://docknetwork.github.io/sdk/tutorials/tutorial_claim_deduction.html)

**Pull request merged**